metrics.py

```diff
@@ -473,27 +455,41 @@ def display_version_slowdown(cls, tasks):
                 raise
 
         # call mincost_path
-        def calculate_maxcost_path_weight(some_task):
+        def calculate_ideal_maxcost_path_weight(some_task):
             ''' helper to pass to graph constructor'''
             timedelta_weight = some_task['finish_time'] - some_task['start_time']
             seconds =  timedelta_weight.total_seconds()
             # invert to allow calculation of maxcost path by mincost-path algo
             return -1 * seconds
-        depgraph = cls(tasks, calculate_maxcost_path_weight)
+        depgraph = cls(tasks, calculate_ideal_maxcost_path_weight)
         idealized_latency = depgraph.depends_on_graph.shortest_paths_dijkstra(
                 source=source_vertex_id, target=target_vertex_id, weights='weight')
         # have to add 1 second to correct for dummy_source second-long runtime,
         # then multiply by -1 again to make the mincost path positive.
         idealized_latency_seconds = (idealized_latency[0][0] +1 ) * -1
 
-        slowdown = real_version_latency_seconds/idealized_latency_seconds
+        def calculate_real_maxcost_path_weight(some_task):
+            ''' helper to pass to graph constructor'''
+            timedelta_weight = some_task['finish_time'] - some_task['begin_wait']
+            seconds =  timedelta_weight.total_seconds()
+            # invert to allow calculation of maxcost path by mincost-path algo
+            return -1 * seconds
+
+        depgraph = cls(tasks, calculate_real_maxcost_path_weight)
```

Comment:
This code is a bit repetitive/boilerplate, but it's short enough and I'm not sure I want to abstract this into a function that simply takes a single time key or two time keys.